### DISCUSSION TITLE
中文书写下省略号是否应该统一标准

### DISCUSSION THREAD

**Discussion by winterwd1201** [sparanoid/chinese-copywriting-guidelines#218] [General]
按照中文书写规范，省略号应使用全角形式，包含六个点（……），而不是三个点（…）。
但是在日常表达中，很多人认为三个点（…）能更好地表达自己的情绪。
请问大家认为有无必要统一中文书写的省略号标准呢？

**Comment by liyishuai** [comment]
据现行标准：
- 中华人民共和国国家标准 GB/T 15834―2011 《标点符号用法》第4.11节“省略号”：
	> 4.11.2 形式
	省略号的形式是“……”。
- 教育部國語推行委員會《重訂標點符號手冊》修訂版：
	> **刪節號　……**
	**位置**　　占行中二格。

正式场合统一使用六点即可。
